app/vmalert/config/log/logger.go

```diff
@@ -0,0 +1,46 @@
+package log
+
+import "github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
+
+// Logger is using lib/logger for logging
+// but can be disabled via Disable method
+type Logger struct {
+	disabled bool
+}
+
+// Disable whether to ignore message logging.
+// Once disabled, logging continues to be ignored
+// until logger is enabled again.
+// Not thread-safe.
+func (l *Logger) Disable(v bool) {
```

Comment:
Wdyt about renaming this to `Supress` or maybe using 2 methods like disable/enable to make it more verbose?
Current naming is a bit confusing when reading:
```
cLogger.Disable(true)
defer cLogger.Disable(false)
```

Reply:
updated